lib/rucio/core/authentication.py

```diff
@@ -282,7 +285,7 @@ def get_ssh_challenge_token(account, appid, ip=None, session=None):
     # Cryptographically secure random number.
     # This requires a /dev/urandom like device from the OS
     rng = random.SystemRandom()
-    crypto_rand = rng.randint(0, sys.maxint)
+    crypto_rand = rng.randint(0, sys.maxsize)
```

Comment:
Not sure about this. sys.maxint is not available in python3. Instead sys.maxsize is still available but not sure if it can be also used here ([https://docs.python.org/2/library/sys.html#sys.maxint](https://docs.python.org/2/library/sys.html#sys.maxint)).

Reply:
Ok, the six package seems to have a constant for this

Reply:
I think this is fine.